bucket/keepass-plugin-readable-passphrase.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.2.1",
+    "description": "Plugin for KeePass 2.x that generates passphrases which are (mostly) grammatically correct but nonsensical.",
+    "homepage": "https://github.com/ligos/readablepassphrasegenerator",
+    "license": "Apache-2.0",
+    "depends": "extras/keepass",
+    "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-1.2.1/ReadablePassphrase.1.2.1.plgx",
+    "hash": "d91261ae0001fd1c78775e8d4354b69bb4b33432811e8e66a4ba19eb185ac35a",
+    "installer": {
+        "script": "Copy-Item \"$dir\\$fname\" \"$(appdir keepass $global)\\current\\Plugins\""
+    },
+    "uninstaller": {
+        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\$fname\""
+    },
+    "checkver": {
+        "github": "https://github.com/ligos/readablepassphrasegenerator"
+    },
+    "autoupdate": {
+        "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-$version/ReadablePassphrase.$version.plgx"
```

Comment:
```suggestion
        "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-$version/ReadablePassphrase.$version.plgx#/ReadablePassphrase.plgx"

```

Reply:
!

Reply:
Whoops! Missed that URL. Good spotting. I've corrected it in my latest commit.